index.js

```diff
@@ -0,0 +1,9 @@
+import {Singular} from "./Singular"
+import {SingularConfig} from "./SingularConfig";
+import {SingularPurchase, SingularIOSPurchase, SingularAndroidPurchase} from "./SingularPurchase"
+
+module.exports.Singular = Singular;
```

Comment:
Not sure you should be using the `module.exports` syntax as that's old `common js`
There's the es6 `export` statement, equivalent to the `import` you use here (I believe there's also an `export from import` variant)